packages/router/src/config.ts

```diff
@@ -51,7 +51,7 @@ export const TChainConfig = Type.Object({
   providers: Type.Array(Type.String()),
   confirmations: Type.Number({ minimum: 1 }),
   defaultInitialGas: Type.Optional(TIntegerString),
-  subgraph: Type.String(),
+  subgraph: Type.Array(Type.String()),
```

Comment:
@LayneHaber changed this to array type, but below I'm checking to see if the subgraph property passed in is a string type - and it handles 'coercing' it into an array, per se. Does that work? It looks like we only do validation at the end there, just wanted to double check that this is functional.